packages/extension/src/routes/Home.vue

```diff
@@ -210,12 +221,39 @@ export default {
       this.$refs.context.open(event, post);
     },
 
+    onPostDndMenu(event) {
+      ga('send', 'event', 'Dnd', 'Menu');
+      this.$refs.dndContext.open(event);
+    },
+
     onPostMenuOpened(event, post) {
       const rect = event.target.getBoundingClientRect();
       this.$refs.context.positionMenu({ bottom: rect.top - 8, right: rect.right });
       this.selectedPostId = post.id;
     },
 
+    onDndMenuOpened(event) {
+      const rect = event.event.target.getBoundingClientRect();
+      this.$refs.dndContext.positionMenu({ top: rect.bottom + 7, right: rect.right });
+    },
+
+    onDisableDndMode() {
+      this.$refs.dndContext.close();
+      this.disableDndMode();
+    },
+
+    enableDndMode(type = 'hour') {
+      let dndDate = new Date();
+      if (type === 'hour') {
+        dndDate.setHours(dndDate.getHours() + 1);
+      } else if (type === 'tomorrow') {
+        dndDate = new Date(dndDate.getFullYear(), dndDate.getMonth(), dndDate.getDate() + 1);
+      }
+
+      this.$refs.dndContext.close();
+      this.$store.commit('ui/setDndModeTime', dndDate.getTime());
```

Comment:
Let's use `mapMutations` to map this mutation as well rather than calling `commit` directly